### DISCUSSION TITLE
카프카에서 partition leader 는 뭘까? 사실상 broker 일까?

### DISCUSSION THREAD

**Discussion by kyungjunleeme** [kyungjunleeme/kafka_study#7] [Q&A] [answered]
모든 파티션은 leader partion이 있고, 원칙상은 leader partition으로부터 데이터를 conusme 해야하니, 일종의 broker라고 생각해도 되지 않을까?

cf) https://stackoverflow.com/questions/60835817/what-is-a-partition-leader-in-apache-kafka
cf) https://www.confluent.io/blog/hands-free-kafka-replication-a-lesson-in-operational-simplicity/

**Comment by psh5487** [comment, marked as answer]
1대의 서버엔 한 개의 브로커가 올라온다고 하는걸보니, 의미상 브로커 = 서버 로 많이 쓰이는 것 같네요
참고로 브로커, 토픽, 파티션의 관계는 요런 식으로 되는 것 같슴당
![스크린샷 2024-12-24 오전 1 13 25](https://github.com/user-attachments/assets/5e52299e-534f-4870-8937-3c48864df0dc)

**Reply by kyungjunleeme** [reply]
@psh5487 오호! 한 번에 이해가되었습니다. 감사합니다.
